commandapi-nms/commandapi-1.19-common/src/main/java/dev/jorel/commandapi/nms/NMS_1_19_Common.java

```diff
@@ -503,45 +468,41 @@ public final org.bukkit.loot.LootTable getLootTable(CommandContext<CommandSource
 	public final ParticleData<?> getParticle(CommandContext<CommandSourceStack> cmdCtx, String str) {
 		final ParticleOptions particleOptions = ParticleArgument.getParticle(cmdCtx, str);
 		final Level level = cmdCtx.getSource().getLevel();
```

Comment:
This `level` variable doesn't get used by anything other than `VibrationParticleOption`. We can remove this and use:

```java
return getParticleDataAsVibrationParticleOption(cmdCtx, cmdCtx.getSource().getLevel(), particle, options);
```